apps/web/src/app/billing/services/billing-notification.service.spec.ts

```diff
@@ -30,6 +30,17 @@ describe("BillingNotificationService", () => {
       });
     });
 
+    it("should use provided title", () => {
```

Comment:
suggestion(non-blocking): clarify where where and how the title is used
```suggestion
    it("shows error toast with the provided error", () => {
```